prodigy/distribution/generating_function.py

```diff
@@ -31,6 +32,97 @@ def _term_generator(function: sympy.Poly):
         poly -= poly.EC() * poly.EM().as_expr()
 
 
+def _parse_to_sympy(expr: Any, real=True, **kwargs) -> sympy.Expr:
```

Comment:
Maybe this should be refactored to a different place than the generating function class, as it is not part of it.